contentcuration/contentcuration/frontend/shared/feedbackApiUtils.js

```diff
@@ -16,6 +16,8 @@ export const FeedbackTypeOptions = {
 // This is mock currently, fixed value of URL still to be decided
 // referencing the url by name
 export const FLAG_FEEDBACK_EVENT_URL = urls[`${'flagged'}_${'list'}`];
+const RECCOMMENDATION_EVENT_URL = 'TBD';
+const RECCOMMENDATION_INTERACTION_EVENT_URL = 'TBD';
```

Comment:
Seems you added the viewsets to Django's URL routing, so the should exist in `urls`. You can probably set them here.